static/admin/config.yml

```diff
@@ -1992,7 +1992,414 @@ collections:
+          - {
+              label: "Programs and Events Card",
+              name: "programsAndEvents",
+              widget: "list",
+              allow_add: true,
+              fields:
+                [
+                  {
+                    label: "Image",
+                    name: "image",
+                    widget: file,
+                    media_library: { config: { multiple: false } },
+                  },
+                  { label: "Title", name: "title", widget: "string" },
+                  {
+                    label: "Description",
+                    name: "description",
+                    widget: "string",
+                  },
+                  { label: "URL", name: "url", widget: "string" },
+                ],
+            }
+          - {
+              label: "Projects and Tools Section",
+              name: "projectsAndToolsInfo",
+              widget: "object",
+              collapsed: true,
+              fields:
+                [
+                  {
+                    label: "Row Reverse",
+                    name: "rowReverse",
+                    widget: boolean,
+                    required: false,
+                    default: false,
+                  },
+                  {
+                    label: "Pre Title",
+                    name: "preTitle",
+                    widget: "string",
+                    minimal: true,
+                  },
+                  {
+                    label: "Title",
+                    name: "title",
+                    widget: "string",
+                    minimal: true,
+                  },
+                  {
+                    label: "Description",
+                    name: "description",
+                    widget: "string",
+                    required: false,
+                  },
+                ],
+            }
+          - {
+              label: "Projects and Tools Card",
+              name: "projectsAndTools",
+              widget: "list",
+              allow_add: true,
+              fields:
+                [
+                  {
+                    label: "Image",
+                    name: "image",
+                    widget: file,
+                    media_library: { config: { multiple: false } },
+                  },
+                  { label: "Title", name: "title", widget: "string" },
+                  {
+                    label: "Description",
+                    name: "description",
+                    widget: "string",
+                  },
+                  {
+                    label: "Categories",
+                    name: "categories",
+                    widget: "list",
+                    allow_add: true,
+                    required: true,
+                    fields: [{ label: Label, name: label, widget: string }],
+                  },
+                  {
+                    label: "Timestamp",
+                    name: "timestamp",
+                    widget: "number",
```

Comment:
Maybe we could try to use `datetime ` here? I'm not sure but maybe we would have to then convert it to timestamp in the code.

Reply:
Resolved in https://github.com/threshold-network/website/pull/94/commits/293e4dc47581a315fea952d9cc2941ac85432e22